save-backend/Backend-API.md

```diff
@@ -0,0 +1,269 @@
+saved into variable `project` by following request:
+
+```bash
+project=$(curl -X GET "${SAVE_CLOUD_URL}/api/v1/projects/get/organization-name?name=save&organizationName=${organizationName}" \
+-H "X-Authorization-Source: ${SAVE_CLOUD_AUTH_SOURCE}" \
+-H "Authorization: ${SAVE_CLOUD_AUTH}")
+```
+Unifying all above, here the examples of how to submit execution, both in
+Git and Standard modes via curl:
+
+<details>
+  <summary>Request for execution submission in Git Mode</summary>
+
+```bash
+curl -X POST "${SAVE_CLOUD_URL}/api/v1/submitExecutionRequest" \
+-H "X-Authorization-Source: ${SAVE_CLOUD_AUTH_SOURCE}" \
+-H "Authorization: ${SAVE_CLOUD_AUTH}" \
+-F "executionRequest={
+    \"project\": ${project},
+    \"gitDto\": {
+        \"url\": ${gitUrl},
+        \"username\": ${gitUserName},
+        \"password\": ${gitPassword},
+        \"branch\": ${branch},
+        \"hash\": ${commitHash}
+    },
+    \"testRootPath\": ${testRootPath},
+    \"sdk\": {
+      \"name\": ${sdkName},
+      \"version\": ${sdkVersion}
+    },
+    \"executionId\" : null
+};type=application/json" \
+-F 'file={
+  "name": "ktlint",
+  "uploadedMillis": 1637658398621,
+  "sizeBytes": 54167132,
+  "isExecutable": false
+};type=application/json' \
+-F 'file={
+  "name": "diktat.jar",
+  "uploadedMillis": 1637658396121,
+  "sizeBytes": 6366668,
+  "isExecutable": false
+};type=application/json'
+```
+
+</details>
+
+<details>
+  <summary>Request for execution submission in Standard Mode</summary>
+
+```bash
+curl -X POST "${SAVE_CLOUD_URL}/api/v1/executionRequestStandardTests" \
+-H "X-Authorization-Source: ${SAVE_CLOUD_AUTH_SOURCE}" \
+-H "Authorization: ${SAVE_CLOUD_AUTH}" \
+-F "execution={
+    \"project\": ${project},
+    \"testsSuites\": ${testSuites},
+    \"sdk\": {
+      \"name\": ${sdkName},
+      \"version\": ${sdkVersion}
+    },
+    \"executionId\" : null
+};type=application/json" \
+-F 'file={
+  "name": "ktlint",
+  "uploadedMillis": 1637658398621,
+  "sizeBytes": 54167132,
+  "isExecutable": false
+};type=application/json' \
+-F 'file={
+  "name": "diktat-analysis.yml",
+  "uploadedMillis": 1637673340431,
+  "sizeBytes":3207,
+  "isExecutable":false
+};type=application/json' \
+-F 'file={
+  "name": "diktat.jar",
+  "uploadedMillis": 1637658396121,
+  "sizeBytes": 6366668,
+  "isExecutable": false
+};type=application/json'
+```
+</details>
+
+These requests will also return execution id, which could be used for getting the
+execution results or for rerun command.
+
+To get execution results, knowing the execution id, use the following request:
+
+```bash
+# Taken after submitExecutionRequest request
+executionId=42
+
+curl -X GET "${SAVE_CLOUD_URL}/api/v1/executionDto?executionId=${executionId}" \
+-H "X-Authorization-Source: ${SAVE_CLOUD_AUTH_SOURCE}" \
+-H "Authorization: ${SAVE_CLOUD_AUTH}"
+```
+
+
+You are also able to get results of latest execution by `/latestExecution` endpoind,
```

Comment:
Please describe the format example of the data returned from endpoint

Reply:
Added